docker/eventmesh-defibus/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM centos7-jdk8:latest
```

Comment:
This image should give some relevant instructions.

Reply:
OK, I'll supply readmes for Dockerfiles.